spinalcordtoolbox/gui/base.py

```diff
@@ -0,0 +1,595 @@
+
+        Returns
+        -------
+        dict        Dictionnary that links the label position, from top to down, to the label definition according to
+                    https://github.com/neuropoly/spinalcordtoolbox/issues/1205
+        """
+        dict = {'1': 50,
+               '2': 49,
+               '3': 1,
+               '4': 3, }
+        for ii in range(5, 30):             # does not matter if the dictionnary is a bit too long. The number of possible labels is still 27.
+            dict[str(ii)] = ii - 1
+        return dict
+
+    def set_data_to_display(self, image, current_point, view):
+        if view == 'ax':
+            self.cross_to_display = [[[current_point.y, current_point.y], [-10000, 10000]],
+                                     [[-10000, 10000], [current_point.z, current_point.z]]]
+            self.aspect_ratio = self.viewer.aspect_ratio[0]
+            return (image.data[current_point.x, :, :])
+        elif view == 'cor':
+            self.cross_to_display = [[[current_point.x, current_point.x], [-10000, 10000]],
+                                     [[-10000, 10000], [current_point.z, current_point.z]]]
+            self.aspect_ratio = self.viewer.aspect_ratio[1]
+            return (image.data[:, current_point.y, :])
+        elif view == 'sag':
+            self.cross_to_display = [[[current_point.x, current_point.x], [-10000, 10000]],
+                                     [[-10000, 10000], [current_point.y, current_point.y]]]
+            self.aspect_ratio = self.viewer.aspect_ratio[2]
+            return (image.data[:, :, current_point.z])
+
+    def show_image(self, im_params, current_point):
+        if not current_point:
+            current_point = Coordinate([int(self.images[0].data.shape[0] / 2), int(self.images[0].data.shape[1] / 2),
+                                        int(self.images[0].data.shape[2] / 2)])
+        for i, image in enumerate(self.images):
+            data_to_display = self.set_data_to_display(image, current_point, self.view)
+            (my_cmap, my_interpolation, my_alpha) = self.set_image_parameters(im_params, i, mpl.cm)
+            my_cmap.set_under('b', alpha=0)
+            self.figs.append(self.axes.imshow(data_to_display, aspect=self.aspect_ratio, alpha=my_alpha))
+            self.figs[-1].set_cmap(my_cmap)
+            self.figs[-1].set_interpolation(my_interpolation)
+            if (self.list_intensites):
+                self.figs[-1].set_clim(self.list_intensites[0], self.list_intensites[1])
+
+    def set_image_parameters(self, im_params, i, cm):
+        if str(i) in im_params.images_parameters:
+            return (
+                copy(cm.get_cmap(im_params.images_parameters[str(i)].cmap)), im_params.images_parameters[str(i)].interp,
+                float(im_params.images_parameters[str(i)].alpha))
+        else:
+            return (cm.get_cmap('gray'), 'nearest', 1.0)
+
+    def remove_axis_number(self):
+        self.axes.set_axis_bgcolor('black')
```

Comment:
according to https://matplotlib.org/devdocs/api/_as_gen/matplotlib.axes.Axes.set_axis_bgcolor.html it's deprecated since 2.0 and as we work on 1.5.3, it should be ok